JSDemos/Demos/DataGrid/RemoteReordering/description.md

```diff
@@ -1,12 +1,13 @@
 This demo shows how to use drag and drop to reorder records stored on the server. This functionality requires that records' order indexes are in an individual data field (`OrderIndex` in this demo) and sorted against that field.
 
-Row drag and drop is configured in the [rowDragging][5] object. Set [allowReordering][2] to **true** to enable this feature. When a row is dropped, the [onReorder][0] event handler is called. Use it to update the record's `OrderIndex` on the server.
+Row drag and drop is configured in the [rowDragging][5] object. Set [allowReordering][2] to **true** to enable this feature. To specify the highlight mode of the row's drop position, use the [dropFeedbackMode][6] property. In this demo, it is set to *"push"*: rows move apart with animation to highlight the new position of a row.
```

Comment:
Can you clarify what you mean by "rows move apart with animation"? If possible, I'd like to see this in action. :)

Reply:
@albertov05 
"rows move apart with animation" in the following demo when a user drags a row: https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/RemoteReordering/jQuery/Light/

Reply:
Thank you. Based on my understanding of the demo, rows seem to move either up or down to create a space to position the row. What do you think about the following suggestion?

`In this demo, it is set to *"push"*: rows move up or down with animation to create space for the new position of the row.`